packages/plugin-core/src/web/utils/site.ts

```diff
@@ -0,0 +1,118 @@
+import {
```

Comment:
nit: to follow the new convention - can you rename this file to the exported class? so `SiteUtilsWeb.ts`